src/state.rs

```diff
@@ -26,6 +26,11 @@ pub struct BulletState {
     pub owned_by_player: bool
 }
 
+pub struct EnemyState {
```

Comment:
Maybe enemies should also have a life property

Reply:
maybe later IMHO ;)